flexmeasures/ui/templates/crud/asset.html

```diff
@@ -7,38 +7,44 @@
 
 
 {% block divs %}
-<nav aria-label="breadcrumb">
-  <ol class="breadcrumb">
-    {% for breadcrumb in breadcrumb_info["ancestors"] %}
-      <li class="breadcrumb-item{% if loop.last %} dropdown active{% endif %}" {% if loop.last %}aria-current="page"{% endif %}>
-        {% if breadcrumb["url"] is not none and not loop.last %}
-          <a href="{{ breadcrumb['url'] }}">{{ breadcrumb['name'] }}</a>
-        {% else %}
-          <a class="dropdown-toggle" data-toggle="dropdown" href="#">{{ breadcrumb['name'] }}<span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            {% for sibling in breadcrumb_info["siblings"] %}
-              <li><a href="{{ sibling['url'] }}">{{ sibling["name"] }}</a></li>
-            {% endfor %}
-          </ul>
-        {% endif %}
-      </li>
-    {% endfor %}
-  </ol>
+
+<nav aria-label="breadcrumb ">
+    <ol class="breadcrumb p-2">
+        {% for breadcrumb in breadcrumb_info["ancestors"] %}
+        <li class="breadcrumb-item{% if loop.last %} dropdown active{% endif %}" {% if loop.last %}aria-current="page"{% endif %}>
+            {% if breadcrumb["url"] is not none and not loop.last %}
+            <a  style="text-decoration:none;" href="{{ breadcrumb['url'] }}">{{ breadcrumb['name'] }}</a>
+            {% else %}
+            <a style="text-decoration:none;" href="{{ breadcrumb['url'] }}" class="dropdown-toggle" data-bs-toggle="dropdown"  aria-expanded="false">{{ breadcrumb['name'] }}
+            </a>
+            <ul class="dropdown-menu">
+                {% for sibling in breadcrumb_info["siblings"] %}
+                <li><a  class="dropdown-item" href="{{ sibling['url'] }}">{{ sibling["name"] }}</a></li>
```

Comment:
I removed `class="dropdown-item"` here and that fixed the reloading bug.

The styling is also influenced and doesn't look as good anymore.

Anyway, this should help you use [Bootstrap 5 Dropwdowns](https://getbootstrap.com/docs/4.0/components/dropdowns/) correctly.

Reply:
I understand that we didn't use the dropwdown items completely by the book before, probably (I don't know how BS3 did it), so the main goal for this PR is to make this work, not do it by the book.

Also, I linked to BS4 docs above, [this is the BS5 link](https://getbootstrap.com/docs/5.3/components/dropdowns/).

Reply:
Both links are for bootstrap 4

Reply:
The second link clearly has "5.3" in the URL.

Reply:
https://getbootstrap.com/docs/5.3/components/dropdowns/